Resfeber/View Controllers/TripsViewController.swift

```diff
@@ -42,27 +45,39 @@ class TripsViewController: UIViewController {
 
         return UIBarButtonItem(customView: button)
     }()
-    
+
     weak var sideMenuDelegate: SideMenuDelegate?
 
     // MARK: - Lifecycle
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
         configureViews()
+        tripService = TripService()
```

Comment:
Delete this line if you decide to initialize tripService directly.
(Refer to inline comment on line 17)
```suggestion
```